src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -430,6 +432,10 @@ public function bulkinsert(Table $table, array $rows): void
                 foreach ($row as $v) {
                     if ($v instanceof Literal) {
                         continue;
+                    } elseif ($v instanceof DateTime) {
+                        $vals[] = $v->toDateTimeString();
```

Comment:
Would doing `str($v);` have a similar effect as doing `$v->toDateTimeString()`?

Reply:
No
Because thats causing it to transform to the string representation, being usually localized (mm/dd/yy) etc